config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -1134,6 +1134,7 @@ presubmits:
         - '--test-args=--container-runtime-endpoint=unix:///var/run/crio/crio.sock --container-runtime-process-name=/usr/local/bin/crio --container-runtime-pid-file= --kubelet-flags="--cgroup-driver=systemd --cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/crio.service --kubelet-cgroups=/system.slice/kubelet.service" --extra-log="{\"name\": \"crio.log\", \"journalctl\": [\"-u\", \"crio\"]}"'
         - --image-config-file=/home/prow/go/src/k8s.io/test-infra/jobs/e2e_node/crio/latest/image-config-cgrpv2-k8s-infra-prow-build-kubetest2.yaml
   - name: pull-crio-cgroupv1-node-e2e-features
+    cluster: k8s-infra-prow-build
```

Comment:
This is the only job that was missing a cluster tag.  All other presubmits have a cluster so they have already been migrated to the community cluster.